docs/src/extend/firmware-management/device-onboarding.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Onboarding a Device
+tags: [Extend, Build]
+sidebar_position: 5
+---
+
+Onboarding a new devices involves configuring thin-edge.io to connect to a cloud for the first time. This generally involves the following steps:
+
+1. Configure the cloud url to connect to
+2. Generate the device certificate (if it does not already exist)
+3. Upload the device certificate to the cloud
+4. Connect the associated tedge-mapper
+
+## Onboarding a device to Cumulocity
+
+The onboarding described in this section involved onboarding a device by executing remote commands on the device via SSH.
+
+It is a two step process which is described as follows:
+
+1. Detect the device (by using DNS Service Discovery)
+2. Onboard it via ssh commands
+
+
+### Pre-requisites
+
+On your machine you will need on of the following:
+
+1. One of the following CLI tools to scan for devices
+
+    * avahi-browse
+    * dns-sd
+
+    :::tip MacOS Users
+    **dns-sd** is installed by default.
+    :::
+
+2. A valid ssh key which matches the authorized key loaded in the built image
```

Comment:
A link to this line would be useful in the how-to build a rugpi image.

Reply:
Resolved by e8a7280cd502a03f328058a993e9407ff2d61448